problem4.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="problem4.css"/>
+    <title>가계부</title>
+</head>
+<body>
+<h1>어디보자 얼마 남았나?</h1>
+<header class="totalsum">
+<h3>호주머니</h3><p><strong> 71000</strong></p>
+<div class="total" >
+<div class="allincome"> <span class="plus-button">+</span> 총수입 150000</div>
+<div class="alloutcome"><span class="minus-button">-</span> 총지출 79000</div>
+</div>
+</header>
+<div class="date">
+<button class="left">&lt;</button> 10월 12일 <button class="right">&gt;</button>
+</div>
+<section class="main">
+내역리스트 
+<div>
+<input type="checkbox" checked="checked"/><button class="income">수입</button>
+<input type="checkbox" checked="checked"/><button class="outcome">수출</button>
```

Comment:
```suggestion
<input type="checkbox" checked/><button class="income">수입</button>
```

checked 속성 그냥 이렇게만 주시면 됩니다!
input에서 checked 속성은 boolean값이에요!

Reply:
오앙 이제 보인다! 안쓰면 false 쓰면 true인 거구나! 
checked 속성은 불리언(boolean) 속성입니다.
불리언 속성은 해당 속성을 명시하지 않으면 속성값이 자동으로 false 값을 가지게 되며, 명시하면 자동으로 true 값을 가지게 됩니다.